src/main/java/com/google/sps/servlets/WaterPollutionServlet.java

```diff
@@ -0,0 +1,121 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.google.sps.servlets;
+
+import java.io.IOException;
+import java.net.URL;
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.Scanner;
+
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+import com.google.gson.Gson;
+import com.google.sps.data.WaterSystem;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+/** Servlet that returns a list of all Superfund Sites */
+@WebServlet("/water")
+public class WaterPollutionServlet extends HttpServlet {
+
+    private static final Logger logger = LoggerFactory.getLogger(WaterPollutionServlet.class);
+    
+    public static final String EPA_WATERSYSTEM_LINK = "https://enviro.epa.gov/enviro/efservice/WATER_SYSTEM/";
+    public static final String EPA_CITY_PARAMETER = "CITIES_SERVED/CONTAINS/";
+    public static final String EPA_STATE_PARAMETER = "/PRIMACY_AGENCY_CODE/";
+    public static final String MIN_DATE = "/ENFDATE/>/01-JAN-14";
+    public static final String CSV_FORMAT = "/Excel/";
+    public static final String SPLITERATOR = "\",\"";
+    public static final int TOTAL_CELL_COUNT = 47;
+
+    private static final String AREA_PARAMETER = "area";
+    private static final String ZIP = "zip";
+    private static final String TOWN_PARAMATER = "town";
+    private static final String STATE_PARAMATER = "state";
+
+    private static final String ZIP_PARAMETER = "zip_code";
+
+    public static final int DEFAULT_SCORE = 100;
+
+    @Override
+    public void init() {
+    }
+
+    @Override
+    public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
+        String area = request.getParameter(AREA_PARAMETER);
```

Comment:
Are you planning to bring back querying by zip code/area parameter in the future? If no, remove the associated code.